elastic/kibana
# [App Search] Add a Sample Engine CTA panel to the engines table when empty

## Summary

Migrates over the Sample Engine CTA from the old repository. This encourages users to get up and running quickly in App Search by using our national parks demo dataset.  

### Screenshots

![image](https://user-images.githubusercontent.com/2479295/111520747-c3e8ee80-872e-11eb-97ab-81f910fbe503.png)

### QA

1. Spin up Kibana and Enterprise Search
2. Log in to Enterprise Search, then App Search, and go to the Engines Overview page
3. Delete any and all existing engines you have, you need to be on the empty state for the Engines Overview page
- [ ] Observe the new CTA as in the screenshot above
4. Click the button
- [ ] Observe the button goes into a loading/spinner state
- [ ] Observe you are brought to Engine Detail page for the `national-parks-demo` engine
- [ ] Observe a success message at the top of the page

### Checklist

Delete any items that are not applicable to this PR.
